src/validator/CreditCardVendor.java

```diff
@@ -0,0 +1,32 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardVendor {
+	private String name;
+	private ArrayList<Integer> numberOfChars;
+	private ArrayList<String> IIN;
+	
+	public String getName(){
+		return name;
+	}
+	
+	public ArrayList<Integer> getNumberOfChars(){
+		return numberOfChars;
+	}
+	
+	public ArrayList<String> getIIN(){
+		return IIN;
+	}
+	
+	public CreditCardVendor(String vendorName, ArrayList<Integer> charsNumbers, ArrayList<String> masks){
```

Comment:
move constructor above getters. It's a good practice to name the contrcutor parameters same as the fields to avoid confusion